app/models/shelter.rb

```diff
@@ -31,4 +32,10 @@ def alphabetical_pets
   def shelter_pets_filtered_by_age(age_filter)
     adoptable_pets.where('age >= ?', age_filter)
   end
+
+  def with_pending_applications
+    if applications.where(status: 'Pending') != nil
+      self.name
```

Comment:
I wonder if we could use a `.pluck(name)` here if there are more than one shelter with pending applications.

Reply:
Or we could try within the method `joins(:applications).where('status = Pending').pluck(shelters.name).distinct